airflow/cli/commands/dag_command.py

```diff
@@ -208,8 +208,8 @@ def dag_state(args):
         dag = get_dag(args.subdir, args.dag_id)
     else:
         dag = get_dag_by_file_location(args.dag_id)
-    dr = DagRun.find(dag.dag_id, execution_date=args.execution_date)
-    print(dr[0].state if len(dr) > 0 else None)  # pylint: disable=len-as-condition
+    dr = DagRun.find(dag.dag_id, execution_date=args.execution_date).one_or_none()
+    print(dr.state if dr else None)  # pylint: disable=len-as-condition
```

Comment:
I think the pylint disable can be removed here as a result of this change

Reply:
Oh yes good spot!